### DISCUSSION TITLE
Cannot use MCC to join in specific server

### DISCUSSION THREAD

**Discussion by Sagittaro** [ORelio/Minecraft-Console-Client#2512] [Q&A]
I try to join in this server 
![image](https://github.com/MCCTeam/Minecraft-Console-Client/assets/51532982/0ca5acfa-e523-41c0-b399-265f79ad44b1)
by using MCC, I copied the Ip address and pasted in the .ini file as follows:
`[Main]
[Main.General]
Account = { Login = "", Password = "-" } 
Server = { Host = "mcfkcs.com:12345" }
AccountType = "microsoft"
Method = "mcc"                             
`
then I launched the program and entered my username, MCC cannot ping this IP like this:
![image](https://github.com/MCCTeam/Minecraft-Console-Client/assets/51532982/86a8f308-0743-4e6b-b98a-5c4d13718bb7)
How can I resolve this problem?

**Comment by breadbyte** [comment]
Please enter the server version in `MinecraftClient.ini` under `MinecraftVersion` to skip pinging the server.

**Reply by Sagittaro** [reply]
Although I set the value of MinecraftVersion and EnableForge, the problem show that:
![image](https://github.com/MCCTeam/Minecraft-Console-Client/assets/51532982/5b09c9e5-2af3-4779-a0b6-87f92639906e)

**Comment by ReinforceZwei** [comment]
Put the port number
```
Server = { Host = "mcfkcs.com", Port = 12345 }
```

**Reply by Sagittaro** [reply]
I tried it, but also cannot link to this server. The MCC message is same as my first reply`s image.Maybe it`s not the MCC problem?
